ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -1650,14 +1707,16 @@ describe('RuntimeConfigurationPanel', () => {
     await pickGpuType(wrapper, 'NVIDIA Tesla T4');
     await pickGpuNum(wrapper, 2);
     await pickMainCpu(wrapper, 8);
-    await pickMainDiskSize(wrapper, MIN_DISK_SIZE_GB);
+    await pickDetachableDiskSize(wrapper, MIN_DISK_SIZE_GB);
 
     await mustClickButton(wrapper, 'Create');
     expect(runtimeApiStub.runtime.status).toEqual('Creating');
-    expect(runtimeApiStub.runtime.gceConfig.gpuConfig.gpuType).toEqual(
```

Comment:
why remove this check?

Reply:
gceConfig itself is null. i can add a check for that